src/main/scala/ArgsParser.scala

```diff
@@ -18,6 +18,7 @@ case class OptFlags(
     trackExts: Boolean = false,
     partStats: Boolean = false,
     partCutoff: Int = 8,
+    vcdOn: Boolean = true,
```

Comment:
VCD should not be on by default

Reply:
Made withVcd to be false by fault in the latest commit